### PR TITLE
Remove limitation of insecure port

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started/release_package.md [hunk 1]
```diff
@@ -28,6 +18,23 @@
       kubeconfig: "path_to_kubeconfig_file" #Enter path to kubeconfig file to enable https connection to k8s apiserver
   ```
   
++ (Option) KubeEdge also supports http connection to Kubernetes apiserver.
```
Comment (kevin-wangzefeng) [inline docs/getting-started/release_package.md]:
```suggestion
+ (Optional) KubeEdge also supports insecure http connection to Kubernetes apiserver for testing, debugging cases.
```

Comment (fisherxu) [reply docs/getting-started/release_package.md]: Done.

**Comment by chendave** [comment]
We don't need this anymore? How this got addressed?

**Comment by fisherxu** [comment]
> We don't need this anymore? How this got addressed?

Just set the default option to https. Http port 8080 is often not exposed in production environments. @chendave

**Pull request merged**